tools/types.ts

```diff
@@ -31,6 +31,9 @@ export interface PackageJson {
   devDependencies?: Record<string, string>;
   peerDependencies?: Record<string, string>;
   exports?: Record<string, string | Partial<{ types: string; import: string; require: string }>>;
+  beachball?: {
```

Comment:
**suggestion:** can we extract these tool specific to separate interface and extend PackageJson ? I mean "beachball" is definitely not a standard package.json property